Dmitriy_Vysotskyy2.0/PageObjects/CartPage.cs

```diff
@@ -0,0 +1,95 @@
+using Dmitriy_Vysotskyy2._0.Models;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using SeleniumExtras.PageObjects;
+using SeleniumExtras.WaitHelpers;
+
+namespace Dmitriy_Vysotskyy2._0.PageObjects;
+
+public class CartPage : BasePage
+{
+    public CartPage(IWebDriver driver) : base(driver)
+    {
+        
+    }
+    
+    [FindsBy(How = How.XPath, Using = "//button[contains(@class,'btn btn-success')]")]
+    [CacheLookup]
+    private IWebElement _btnPlaceOrder;
+    
+    [FindsBy(How = How.Id, Using = "name")]
+    [CacheLookup]
+    private IWebElement _txtName;
+    
+    [FindsBy(How = How.Id, Using = "country")]
+    [CacheLookup]
+    private IWebElement _txtCountry;
+    
+    [FindsBy(How = How.Id, Using = "city")]
+    [CacheLookup]
+    private IWebElement _txtCity;
+    
+    [FindsBy(How = How.Id, Using = "card")]
+    [CacheLookup]
+    private IWebElement _txtCard;
+    
+    [FindsBy(How = How.Id, Using = "month")]
+    [CacheLookup]
+    private IWebElement _txtMonth;
+    
+    [FindsBy(How = How.Id, Using = "year")]
+    [CacheLookup]
+    private IWebElement _txtYear;
+    
+    [FindsBy(How = How.XPath, Using = "//button[contains(@class,'btn btn-primary') and text()='Purchase']")]
+    [CacheLookup]
+    private IWebElement _btnPurchase;
+    
+    public void DeleteItem(string itemName)
+    {
+        var lnkDeleteItem=  _wait.Until(ExpectedConditions
+            .ElementExists(By.XPath($"//td[text()='{itemName}']/parent::tr//a")));
+        
+        lnkDeleteItem.Click();
+    }
+
+    public bool CheckDeleteStatus(string itemName)
+    {
+        // _wait.Until(ExpectedConditions
+        //     .ElementExists(By.XPath($"//th[text()='Price']")));
+        Thread.Sleep(2000);
```

Comment:
to avoid using thread sleep you can try using _wait.Until(ExpectedConditions
            .InvisibilityOfElementLocated(By.XPath($"//td[text()='{itemName}']/parent::tr//a")));